File: 3194-find-words-containing-character/find-words-containing-character.py
class Solution(object):
    def findWordsContaining(self, words, x):
        result = []
        for i in range(0, len(words)):
            print(i)
            for j in words[i]:
                print(j)
                if j == x:
                    result.append(i)
                    break
        return result

File: 3194-find-words-containing-character/find-words-containing-character.js
/**
 * @param {string[]} words
 * @param {character} x
 * @return {number[]}
 */
var findWordsContaining = function(words, x) {
    let result = [];
    for(let i=0; i<words.length; i++){
        for(word of words[i]){
            if(x === word){
                result.push(i);
                break;
            }
        }
    }
    return result;
};